src/main/java/frc/robot/commands/AlgeaKnockoffPrepSeqeunce.java

```diff
@@ -0,0 +1,27 @@
+package frc.robot.commands;
+
+import frc.robot.subsystems.arm.Arm;
+import frc.robot.subsystems.arm.Arm.ArmPosition;
+import frc.robot.subsystems.climb.Climb;
+import frc.robot.subsystems.climb.Climb.ClimbPosition;
+import frc.robot.subsystems.elevator.Elevator;
+import frc.robot.subsystems.elevator.Elevator.ElevatorHeight;
+import edu.wpi.first.wpilibj2.command.ConditionalCommand;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+
+public class AlgeaKnockoffPrepSeqeunce extends SequentialCommandGroup  {
+    public AlgeaKnockoffPrepSeqeunce(Elevator elevator, Arm arm, boolean isHigh) {
+        addRequirements(elevator, arm);
+        addCommands(
+            new ConditionalCommand(
+                new SequentialCommandGroup(
+                new InstantCommand(() -> elevator.setDesiredPosition(ElevatorHeight.SCORE_L2)),
+                new InstantCommand(() -> arm.setDesiredPosition(ArmPosition.L2))
```

Comment:
first need to check if deep climb is in interference zone. if it is, then you need to move it out of the way before moving the arm